index.html

```diff
@@ -46,8 +46,169 @@ <h2>I'm a Software Developer</h2>
             </div>
             </div>
         </div>
-        <div class="shapels">
-            <img src="imakges/Illustration (1).png" alt="">
+    </section>
+    <section>
+        <div class="work-section">
+            <div class="recent-work">
+                <h1>My Recent Work</h1>
+                <hr>
+            </div>
+            <div class="project-box">
+                <img src="images/Img Placeholder.png" alt="">
+                    <h2>Multi-Post-Stories</h2>
+                    <p>
+                        A daily selection of privately personalized 
+                        <br> reads; no accounts or sign-ups required.
+                        <br> has been the industry's standard dummy
+                         text ever since the 1500s, when an 
+                        <br> unknown printer took a standard dummy
+                        <br> text.
+                    </p>
+                    <div class="btn">
+                        <ul>
+                            <li>CSS</li>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="see_more_btn">
+                        <button>See Projects</button>
```

Comment:
- [ ] Kindly, make sure that you implement the correct button interactions just as shown on the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=bZX5VS91nNO9rtwR-0). Currently, there's no change on any of the buttons when hover, pressed, etc. As this is part of the project's requirements.
![image](https://user-images.githubusercontent.com/86069740/236590867-d6a8a867-1ea6-4e1e-8cc6-44b15d9abe0d.png)

## Figma
![image](https://user-images.githubusercontent.com/86069740/236590710-270fa9e7-7bcc-443c-8b00-79cc9684744b.png)